pkgs/applications/emulators/yuzu/generic.nix

```diff
@@ -45,6 +45,10 @@
 stdenv.mkDerivation rec {
   inherit pname version src;
 
+  # Replace icons licensed under CC BY-ND 3.0
+  # with free ones to allow for binary redistribution
+  patches = [ ./yuzu-free-icons.patch ];
```

Comment:
It would be nice if you could link to the pull request with this patch, could also explain why we need to use a locally generated copy.